cfn-resources/project/cmd/resource/mappings.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resource
+
+import (
+	"sort"
+
+	"go.mongodb.org/atlas-sdk/v20231115008/admin"
+)
+
+func NewResourceTags(tags map[string]string) []admin.ResourceTag {
+	sliceTags := make([]admin.ResourceTag, 0, len(tags))
+	for _, k := range sortStringMapKeys(tags) {
```

Comment:
[q] if the sorting logic is only needed for testing purposes, do you think we could move it within the unit test assertion logic?

Reply:
🤔 I think you are right; the API doesn't care if the tags are sorted or not. I'll move it to the tests

Reply:
Moved in 9cd869b013b9671f877f98137c995fe603607ad4. Feel free to re-open